LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -618,9 +620,20 @@ - (void)_dismissMe {
 #pragma mark - UI setup
 - (void)_setupNavBarWithLogoutTitle:(NSString *)logoutTitle {
     // Navigation Bar with custom UI
-    self.navBar =
-    [[UINavigationBar alloc] initWithFrame:CGRectMake(0, LTHMainWindow.frame.origin.y,
-                                                      LTHMainWindow.frame.size.width, 64)];
+    if (LTHiOS11) {
+        UIView *patchView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, LTHMainWindow.frame.size.width, [LTHPasscodeViewController getStatusBarHeight])];
+        patchView.backgroundColor = [UIColor whiteColor];
+        [self.view addSubview:patchView];
```

Comment:
Since the `navBar` is added to the window, so should the patch. But, I think it looks better if both are actually added to `self.view`, so if you could change that on the last line of this method, that would be awesome!